cli/common/utils.go

```diff
@@ -143,3 +145,95 @@ func ParseYmlFile(path string) (map[string]interface{}, error) {
 
 	return res, nil
 }
+
+func readFromPos(f *os.File, pos int64, buf *[]byte) error {
+	if _, err := f.Seek(pos, io.SeekStart); err != nil {
+		return fmt.Errorf("Failed to seek: %s", err)
+	}
+
+	if _, err := f.Read(*buf); err != nil {
+		return fmt.Errorf("Failed to read: %s", err)
+	}
+
+	return nil
+}
+
+// GetLastNLinesBegin return the position of last n lines begin
+func GetLastNLinesBegin(filepath string, n int) (int64, error) {
```

Comment:
BTW, have you measure performance of this implementation on big files? Say, 1GB and more.